apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -45,6 +46,12 @@ type ClusterResourcePlacement struct {
 
 // ClusterResourcePlacementSpec defines the desired state of ClusterResourcePlacement.
 type ClusterResourcePlacementSpec struct {
+	// The number of old snapshots to retain to allow rollback.
+	// This is a pointer to distinguish between explicit zero and not specified.
+	// Defaults to 10.
+	// +optional
+	RevisionHistoryLimit *int32 `json:"revisionHistoryLimit,omitempty"`
+
```

Comment:
This field should appear after rolloutstrategy. It is the least important field in this spec.